### PR TITLE
Move default_config.yaml to python defaults module

### PR DESCRIPTION
**Pull request by petr-balogh**

### CODE REVIEW

**Comment by petr-balogh** [comment]
@zmc @vasukulkarni @clacroix12  this was discussed on last automation call that we will get rid of default_config.yaml file.

Also I am moving all usages of ROOK_CLUSTER_NAMESPACE and using cluster_namespace from config.ENV_DATA which should corresponding all the time with one from config.

I think that it should also fix issue #125 .

Anyway I think that in defaults.py we still had a lot of constants so I moved them there.

Struggling with verification cause broken leftovers check which failing tests and also some other odds errors which I see even in master. Also hit the issue https://github.com/red-hat-storage/ocs-ci/issues/294 while verification which also caused a lot of problems. So still need to proper verify.

File: ocs/defaults.py [hunk 1]
```diff
@@ -7,19 +7,47 @@
...
-CLIENT_VERSION = INSTALLER_VERSION
-ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
-KUBECONFIG_LOCATION = 'auth/kubeconfig'  # relative from cluster_dir
-API_VERSION = "v1"
-CEPHFILESYSTEM_NAME = 'ocsci-cephfs'
+
+#############################################################################
+# Default Configuration - access those vars from config and not from here!  #
+#############################################################################
+ENV_DATA = {
```
Comment (vasukulkarni) [inline ocs/defaults.py]: Instead of constants, As per some discussion last week, I think few said they belong to *defaults* and many of the current entries in *defaults* needs a new file name.

File: conf/README.md [hunk 2]
```diff
@@ -81,12 +81,18 @@ from ocs import defaults
 
 # From you code you can access those data like
 
-Taking data from ENV_DATA will always use right cluster_namespace passed via
-`--ocsci-conf` config file or default one defined in `default_config.yaml`.
-function_which_taking_namespace(namespace=config.ENV_DATA['cluster_namespace'])
+# Taking data from ENV_DATA will always use right cluster_namespace passed via
+#`--ocsci-conf` or `--cluster-conf` config files.
+
+# It's not possible to access config data from module level as config is not
```
Comment (vasukulkarni) [inline conf/README.md]: I would reword since its conveying different meaning, maybe just "config data should not be accessed at module level and instead should be used within function definitions or fixtures"

**Comment by zmc** [comment]
Seems we're still not quite in the same page. Several of us met last week and [discussed this](https://github.com/red-hat-storage/ocs-ci/pull/218#pullrequestreview-252938640)

Let's cover this in tomorrow's meeting!

**Comment by petr-balogh** [comment]
> Seems we're still not quite in the same page. Several of us met last week and [discussed this](https://github.com/red-hat-storage/ocs-ci/pull/218#pullrequestreview-252938640)
> 
> Let's cover this in tomorrow's meeting!

things why I wanted to get rid of yaml file are:
- I thought that we discussed this last week probably on our different mtg that you had later on with Vasu
- that ppl are confused and trying to put this default config to --ocsci-conf as a default config if there won't be such file ppl won't import it
- we don't need to have extra layer of loading yaml file if it will be in python module
- we can use some python expressions if needed
- values can be accessed directly if needed from some reason

If ppl want to keep the defautl_config.yaml I can change the PR to keep it and will have just the rest of changes which take care about using namespace from config instead of defaults.py and move the rest of constant values to constants and deleting some unused vars.

File: ocs/defaults.py (hunk 1 above)
Comment (petr-balogh) [reply ocs/defaults.py]: @vasukulkarni not sure I follow here. I can move this config related default to diff module, or we can decide to have it still in yaml file, see my comment below. And discuss probably tomorrow.

File: conf/README.md (hunk 2 above)
Comment (petr-balogh) [reply conf/README.md]: done in next commit

**Comment by clacroix12** [comment]
@petr-balogh is this something we want to move forward with or should we close?

**Comment by petr-balogh** [comment]
> @petr-balogh is this something we want to move forward with or should we close?

Lowered priority here as I dealt with more priority tasks.

I will close this one and will open new one PR later on.
